content/courses/offline-transactions/durable-nonces.md

```diff
@@ -14,39 +14,39 @@ description: "Use durable nonces to sign transactions ahead of time."
   have an expiration date of 150 blocks (~80-90 seconds).
 - After signing a durable transaction you can store it in a database or a file
   or send it to another device to submit it later.
-- A durable transactions is made using a nonce account. A nonce account holds
+- A durable transaction is created using a nonce account. A nonce account holds
   the authority and the nonce value which replaces the recent blockhash to make
   a durable transaction
 - Durable transactions must start with an `advanceNonce` instruction, and the
-  nonce authority has to be a signer in the transaction.
-- If the transaction fails for any reason other than the nonce advanced
-  instruction the nonce will still get advanced, even though all other
-  instruction will get reverted.
+  nonce authority must be a signer of the transaction.
+- If the transaction fails for any reason other than the `advanceNonce`
+  instruction, the nonce will still be advanced, even though all other
+  instructions will be reverted.
 
 ## Overview
 
 Durable Nonces are a way to bypass the expiration date of regular transactions.
-To understand that better, we'll start by looking at the concepts behind regular
-transactions.
+To understand this more clearly, we'll start by looking at the concepts behind
+regular transactions.
 
-In Solana, transactions are made of three main parts:
+In Solana, transactions consist of three main parts:
 
-1. **Instructions**: Instructions are the operations that you want to perform on
-   the blockchain, like transferring tokens, creating accounts, or calling a
+1. **Instructions**: Instructions are the operations you want to perform on the
+   blockchain, like transferring tokens, creating accounts, or calling a
    program. These are executed in order.
 
-2. **Signatures**: Signatures are the proof that the transaction was signed by
-   the required singers/authorities. For instance, if you are transferring SOL
-   from your wallet to another, you'll need to sign the transaction so the
-   network can verify that the transaction is valid.
+2. **Signatures**: Signatures are proof that the transaction was signed by the
+   required signers/authorities. For instance, if you are transferring SOL from
+   your wallet to another, you'll need to sign the transaction so the network
+   can verify that the transaction is valid.
```

Comment:
This is a criticism of the old writing, but the logic here is circular and bad. Signatures prove I sign it, which means I have a signature, which proves I signed it. 

Try:

> **Signatures**: Signatures are proof that the transaction was made by the required signers (assuming they have kept their secret keys secure).